### ISSUE TITLE
Autoresize with min-height from element it is based on

### ISSUE THREAD

**Issue by tomhillgreyridge** [tinymce/tinymce#9350]
## 📝 Provide detailed reproduction steps (if any)
We have lots of places where we have optional notes fields within a web app.  Since it looks a bit daft for them to be tiny, we set a min height on them and this differs based on where it is in the web app (so a forum note might be 200px high and a project summary 500px high).

I'd like to use the autoresize plugin but have the min-height set to match the element it is based on rather than having to hard code it to a fixed height.  The target behaviour would be: 

1. The default height of the editor matches the height of the textarea it is based on, unless the content is longer than that
2. If the content is initially longer than the textarea min-height, the editor expands to match the content height
3. If the user types more content it expands to match the content height
4. If the user deletes content it shrinks, but only to the min-height of the textarea

I tried setting both height and min-height on the textarea, but it doesn't seem to apply when the autoresize plugin is enabled - see

https://fiddle.tiny.cloud/9eWSiel58U/0

Given the min-height of the textarea the editor is based on is 400px, I would like the min-height of the editor to be 400px.  It should be able to extend past that, but not shrink to less than that.

I know I can use min_height: 400 in the init, but that requires a separate config for every single notes field in the application whereas I was hoping to use a single standardized configuration and have it simply match the target element height.

### ✔️ Expected result
See above.

### ❌ Actual result
If autoresize plugin is enabled, it ignores the min-height on the textarea.

### ❓ Possible solution
If there isn't some combination of autoresize plugin options I am missing, is there a way to change the min-height dynamically after the editor has loaded?  If so, we can read the height from the target element and set if after loading.

**Comment by TheSpyder** [comment]
Autoresize does support a min_height configuration:
https://www.tiny.cloud/docs/tinymce/latest/autoresize/#min_height

If you add a `setup` function to your config, that will run before any plugins are initialised.
https://www.tiny.cloud/docs/tinymce/latest/events/#editor-core-events

So you might be able to use `editor.options.set()` in a setup function to adjust the `min_height` config based on the element?
https://www.tiny.cloud/docs/tinymce/latest/apis/tinymce.editoroptions/

**Comment by tomhillgreyridge** [comment]
Yep.  I've got round it by setting up tinymce using target: element and then calculating the element's height and passing it in manually as a min_height.  It would be a nice enhancement to have a min_height: "auto" option on the autoresize plugin which basically means "don't go any smaller than the element it is based on". 

That way, if you want a field which is always 200px at minimum, you just set "min-height: 200px" on the textarea that tinymce is based on and the autoresize plugin would respect that.
